CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - The GitHub PR template is now named `./github/PULL_REQUEST_TEMPLATE.md`
 - Now specify package `requests-2.31.0` in `environment.py` (fixes a security issue)
 - Updated badge links in `README.md`
+- Pass `label=None` in `run_benchmark.py` to shorten operation budget filenames
```

Comment:
Wouldn't this also remove the description of what the table contains from the table itself? I think have that info with the times is useful to have in the file. How about keeping label but removing use of it in the filename?

Reply:
Now fixed in8b3b9c